sdks/python/apache_beam/transforms/external_test.py

```diff
@@ -193,6 +194,20 @@ def test_java_expansion(self):
 
         assert_that(res, equal_to(['a: 3', 'b: 1', 'c: 2']))
 
+      # Test GenerateSequence Java transform
+      options._all_options['shutdown_sources_on_final_watermark'] = True
+      options._all_options['parallelism'] = 1
+      options._all_options['streaming'] = True
+
+      with beam.Pipeline(options=options) as p:
```

Comment:
Can we also add an integration test for GenerateSequence (Java transform from a Python pipeline) for Flink ?

Reply:
Yes, that make sense. We have `ReadSourcePortableTest` but that one is not a full integration test.